### PR TITLE
IOSXE/show ap tag summary

### LINKED ISSUE #138: Cisco IOS show_arp calls super() and does not pass "output" kwarg
It looks to me the call to super() on line 49 of the Cisco IOS show_arp.py file should have the "output" kwarg passed on to the base class but currently this is missing.

### PR DESCRIPTION
**Pull request by robschn**
## Description
Adds parser for the command `show ap tag summary`.

## Motivation and Context
Converts raw output of `show ap tag summary` into structured data.

## Impact (If any)
None

## Screenshots:

image

## Checklist:

- [x] I have updated the changelog.
- [x] I have updated the documentation (If applicable).
- [x] I have added tests to cover my changes (If applicable).
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

**Review by badalsoyantar — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 1]
```diff
@@ -2662,3 +2662,101 @@ def filter_lines(raw_output, remove_lines):
...
+        ap_info_capture = re.compile(
+            r"^(?P<ap_name>\S+)\s+(?P<ap_mac>\S{4}\.\S{4}\.\S{4})\s+(?P<site_tag_name>\S+)\s+(?P<policy_tag_name>\S+)\s+(?P<rf_tag_name>\S+)\s+(?P<misconfigured>\S+)\s+(?P<tag_source>\S+)$"
+        )
+
+        ap_info_obj = {}
+
+        for line in out.splitlines():
+            line = line.strip()
+
+            if ap_number_capture.match(line):
```
[the remark this answers is not in the document]
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_ap.py]: Updated!

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 2]
```diff
@@ -2662,3 +2662,101 @@ def filter_lines(raw_output, remove_lines):
...
+                match = ap_number_capture.match(line)
+                group = match.groupdict()
+
+                # format str to int
+                group["number_of_aps"] = int(group["number_of_aps"])
+
+                ap_info_obj.update(group)
```
[the remark this answers is not in the document]
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_ap.py:2738]: Updated!

File: src/genie/libs/parser/iosxe/tests/ShowApTagSummary/cli/equal/golden_output_expected.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
+expected_output = {
+    "ap_name": {
+        "b25a-13-cap10": {
+            "ap_mac": "3c41.0fee.5094",
+            "site_tag_name": "default-site-tag-fabric",
+            "policy_tag_name": "PT_Fabri_B25_B25-1_fe778",
+            "rf_tag_name": "Standard",
+            "misconfigured": False,
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowApTagSummary/cli/equal/golden_output_expected.py:8]: can take the value `No` as is.
```
"misconfigured": "No",

**Review by tahigash — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/tests/ShowApTagSummary/cli/equal/golden_output_expected.py (hunk 3 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowApTagSummary/cli/equal/golden_output_expected.py]: Updated!

**Review by badalsoyantar — APPROVED** [review]

**Review by tahigash — APPROVED** [review]

**Pull request merged**